File: src/solutions/2023/13/c/main.c
#include "main.h"
#include "run.h"
#include "utils_str.h"
#include "utils_num.h"
#include <stdlib.h>
#include <string.h>

static char *part1(const char *restrict input, const int32_t isTest) {
   return num_tostr(0);
}

static char *part2(const char *restrict input, const int32_t isTest) {
   return num_tostr(0);
}

int main(int argc, char *argv[]) {
   solution_options_t options = getoptions();
return run(argc, argv, part1, part2, options);
}

File: src/solutions/2023/13/ocaml/main.ml
let options = Hashtbl.create 128;;

Hashtbl.add options "has_alternate" false;;
Hashtbl.add options "has_io" false;;

let part_1 (input : string) (_is_test : bool) : string = input
let part_2 (input : string) (_is_test : bool) : string = input
let () = Aoc.run Sys.argv part_1 part_2 options
